src/main/java/com/minecraftabnormals/endergetic/core/mixin/EnderCrystalItemMixin.java

```diff
@@ -46,9 +46,9 @@ private void onItemUse(ItemUseContext context, CallbackInfoReturnable<ActionResu
 							dragonfightmanager.tryRespawnDragon();
 						}
 					}
+					context.getItem().shrink(1);
+					info.setReturnValue(ActionResultType.func_233537_a_(world.isRemote));
```

Comment:
`world.isRemote` gets used above again when testing to spawn the ender crystal.
Now that it's getting used twice, `world.isRemote` should be stored in a local variable.